www/nginx/src/opnsense/mvc/app/controllers/OPNsense/Nginx/forms/upstream.xml

```diff
@@ -20,7 +20,7 @@
   <field>
     <id>upstream.tls_enable</id>
     <label>Enable TLS (HTTPS)</label>
-    <help>Authenticate on the Server using a client certificate.</help>
+    <help>Use TLS (HTTPS) to connect to the Server.</help>
```

Comment:
```suggestion
    <help>Use TLS (HTTPS) to connect to the server.</help>
```

Reply:
I was sticking to convention per example 😝

Reply:
no worries, thanks for the fix 👍